### ISSUE TITLE
ngf-select='condition' and ngf-drop='condition' does not work on v7.0.6

### ISSUE THREAD

**Issue by Mistrall** [danialfarid/ng-file-upload#952]
Hello danail!

We've just upgraded to version 7.0.6 from 5.2.0 and conditional ngf-drop='condition' and ngf-select='condtion' stopped working!
Reproducible on your demo site -> https://angular-file-upload.appspot.com/.
1) Go to demo site
2) Click '+ edit upload html' link
3) Find part of markup
```html
<legend>Play with options</legend>
        <div class="up-buttons">
          <div ngf-select="" ngf-drop="" ....
````
4) Add condition to directives
```html
<legend>Play with options</legend>
        <div class="up-buttons">
          <div ngf-select="false" ngf-drop="false"
```
5) Try to select file via upload box - it still allows to do this but should not

**Comment by danialfarid** [comment]
Fixed at version 7.0.8

**Comment by danialfarid** [comment]
There are two other directives for this now `ngf-select-disabled` and `ngf-drop-disabled`
